Add page for show and info (#13327)
canonical/lxd: doc/explanation/lxc_show_info.md

(lxc-show-info)=

> # About `lxc` 'show' and 'info'

Reviewer: ```suggestion
# About `lxc` `show` and `info`

```

We display the sub-commands in the same way as the main command, thus in code style. That's a global change for this page.

Also, an empty line after the heading, just for consistency. :)

> For the entities managed by LXD, the `lxc` command provides a 'list' sub-command, and may provide 'show' and 'info' sub-commands.

Reviewer: ```suggestion
For the entities managed by LXD, the `lxc` command provides a 'list' sub-command, and might provide 'show' and 'info' sub-commands.
```

"may" is not wrong in this context. But in technical documentation, we're always trying to be as precise as possible and not leave anything up to interpretation - and "may" can mean a lot of different things.
A common way to distinguish is:

- "may" is for permission: You may enter the house (you're allowed to).
- "might" is for possibility: This might happen (or it might not).
- "can" is for ability: You can walk (unless you're in a wheel chair).

The purpose of the 'info' sub-command is to show current state information, and the purpose of the 'show' sub-command is to show configuration information and how the entity is used by other entities.

> For example, the `lxc network info` command shows IP address and traffic statistics;

Reviewer: ```suggestion
For example, the `lxc network info` command shows IP address and traffic statistics:
```

    Name: lxdbr0
    MAC address: 00:16:3e:d3:ec:41
    MTU: 1500
    State: up

    Ips:
      inet    10.247.40.1
      inet6   fd42:d389:f4a1:53d2::1
      inet6   fe80::216:3eff:fed3:ec41

    Network usage:
      Bytes received: 127.66kB
      Bytes sent: 15.54kB
      Packets received: 1433
      Packets sent: 175

> and the `lxc network show` command shows further information, and which entities are using the network:

Reviewer: ```suggestion
The `lxc network show` command, on the other hand, shows how the network is configured, and which entities are using the network:
```

Two comments here:
- You shouldn't continue a sentence if you have code blocks in the middle of it. This works okay in English, but it might cause problems if the documentation is ever translated into other languages where this is not possible.
- I suggest to add in "on the other hand" to make it a bit clearer what we're trying to say in this page - we want to show the difference between the two commands.

(lxc-show-info)=
# About `lxc` `show` and `info`

For the entities managed by LXD, the `lxc` command provides a 'list' sub-command, and might provide `show` and `info` sub-commands.
The purpose of the `info` sub-command is to show current state information, and the purpose of the `show` sub-command is to show configuration information and how the entity is used by other entities.

For example, the `lxc network info` command shows IP address and traffic statistics:

    Name: lxdbr0
    MAC address: 00:16:3e:d3:ec:41
    MTU: 1500
    State: up

    Ips:
      inet    10.247.40.1
      inet6   fd42:d389:f4a1:53d2::1

>       inet6   fe80::216:3eff:fed3:ec41

Reviewer: This is a link-local address so I think it can stay as it is (I'm confirming with the devs).
